### ISSUE TITLE
HOW WILL DARKNET BE AFFECTED BY THE RECENT CRYPTOCURRENCY CRASH??

### ISSUE THREAD

**Issue by pjreddie** [pjreddie/darknet#386]
WE NEED ANSWERS!

**Comment by pjreddie** [comment]
what do you expect, bitcoin and darknet are both fiat currencies with no intrinsic value. at least you can eat gold

**Comment by Grabber** [comment]
nothing has 'intrinsic value'.

**Comment by pjreddie** [comment]
Excuse me, but the U.S. Dollar is also a fiat currency. It has value because people believe in it. Just wait for the market to stabilize and buy into the price drop.

**Comment by Grabber** [comment]
Deleting posts now? "It has value because people believe in it", you gave the answer.

**Comment by pjreddie** [comment]
I knew Darknet was doomed when I heard my doorman talking about it to the valet. So many people using deep learning without understanding the underlying technology!

**Comment by pjreddie** [comment]
GUYS DARKNET IS BELOW 3,000 FORKS, FORK IT NOW BEFORE IT GOES BACK UP

**Comment by pjreddie** [comment]
Is it just me or is there something very fishy about this whole darknet thing? I mean, why store all my models on one central server run by some random graduate student? And now they are saying no one can download models at all to avoid volatility in the training process?

**Comment by pjreddie** [comment]
I ignored warnings about darknet to my peril. IMO the hard fork lightnet is the best option. Speaking of hard forks, what ever happened with the TensorSlow / TensorFlow split?

**Comment by pjreddie** [comment]
WHY IS THIS GUY DELETING POSTS?? WHAT ARE YOU HIDING

**Comment by AlexeyAB** [comment]
Time to Darknet ICO crowdfunding (no shares to avoid problems with https://www.sec.gov/ ).

**Comment by baristahell** [comment]
How about using THE INCREDIBLE DARKNET (TM) to sell cryptococaine to teenagers, get them addicted to DEEP ADVERSARIAL HALLUCINATIONS so you can sell more DARKNETS (TM) ?

tl;dr : notice me senpai

**Comment by smile-on** [comment]
On cryptocurrency crash - "where is the will there's a way".
